### PR TITLE
fix(components/layout): help inline modern theme styles follow design guidelines

### PR DESCRIPTION
**Pull request by Blackbaud-TrevorBurch**
[AB#2294181](https://dev.azure.com/blackbaud/f565481a-7bc9-4083-95d5-4f953da6d499/_workitems/edit/2294181)

### CODE REVIEW

**Comment by blackbaud-sky-build-user** [comment]
[Storybook preview](https://blackbaud.github.io/skyux-pr-preview/845/storybook/)

Component Storybooks:

- [ag-grid-storybook](https://blackbaud.github.io/skyux-pr-preview/845/storybooks/ag-grid-storybook/)
- [indicators-storybook](https://blackbaud.github.io/skyux-pr-preview/845/storybooks/indicators-storybook/)
- [layout-storybook](https://blackbaud.github.io/skyux-pr-preview/845/storybooks/layout-storybook/)
- [modals-storybook](https://blackbaud.github.io/skyux-pr-preview/845/storybooks/modals-storybook/)
- [pages-storybook](https://blackbaud.github.io/skyux-pr-preview/845/storybooks/pages-storybook/)
- [popovers-storybook](https://blackbaud.github.io/skyux-pr-preview/845/storybooks/popovers-storybook/)
- [split-view-storybook](https://blackbaud.github.io/skyux-pr-preview/845/storybooks/split-view-storybook/)
- [tabs-storybook](https://blackbaud.github.io/skyux-pr-preview/845/storybooks/tabs-storybook/)
- [tiles-storybook](https://blackbaud.github.io/skyux-pr-preview/845/storybooks/tiles-storybook/)

Apps:

- [code-examples](https://blackbaud.github.io/skyux-pr-preview/845/code-examples/)
- [dep-graph](https://blackbaud.github.io/skyux-pr-preview/845/dep-graph/)
- [integration](https://blackbaud.github.io/skyux-pr-preview/845/integration/)
- [playground](https://blackbaud.github.io/skyux-pr-preview/845/playground/)

File: libs/components/indicators/src/lib/modules/help-inline/help-inline.component.scss [hunk 1]
```diff
@@ -15,20 +17,23 @@
   }
 }
 
-:host-context(.sky-control-help) {
```
Comment (Blackbaud-TrevorBurch) [inline libs/components/indicators/src/lib/modules/help-inline/help-inline.component.scss:18]: Removed this based on a conversation with @Blackbaud-AdamFunderburk. We want all help inline buttons to have the same padding no matter where they are being constructed. This was removing the padding in places like Box and Tiles previously.

File: libs/components/indicators/src/lib/modules/help-inline/help-inline.component.scss [hunk 2]
```diff
@@ -15,20 +17,23 @@
...
-  .sky-help-inline {
-    padding: 0;
-  }
-}
-
 @include mixins.sky-theme-modern {
   .sky-help-inline {
+    border-radius: 3px;
```
Comment (Blackbaud-TrevorBurch) [inline libs/components/indicators/src/lib/modules/help-inline/help-inline.component.scss:23]: Overwrite the button's default 6px border radius based on design specification.

File: libs/components/indicators/src/lib/modules/help-inline/help-inline.component.scss [hunk 3]
```diff
@@ -15,20 +17,23 @@
...
     ::ng-deep .fa-stack-1x {
       font-size: 10px;
     }
+
+    @include modernMixins.sky-theme-modern-button-variant(
+      $sky-text-color-action-primary,
+      transparent,
+      transparent,
+      $sky-theme-modern-background-color-primary-dark,
+      0px 5px
```
Comment (Blackbaud-TrevorBurch) [inline libs/components/indicators/src/lib/modules/help-inline/help-inline.component.scss:39]: We want the padding to be 1px top/bottom and 5px left/right; however, the mixin adds 1px to the values passed in and so 0px and 5px are specified here.

File: libs/components/theme/src/lib/styles/_public-api/themes/modern/_compat/_mixins.scss [hunk 4]
```diff
@@ -54,8 +54,7 @@
 
 @mixin sky-theme-modern-btn-disabled($color, $background, $border) {
   &.sky-btn-disabled,
-  &[disabled],
-  fieldset[disabled] & {
```
Comment (Blackbaud-TrevorBurch) [inline libs/components/theme/src/lib/styles/_public-api/themes/modern/_compat/_mixins.scss:58]: This rule was not working at all and was causing issues when used inside the `host-context` of the modern theme mixin. Removing and will file a bug.

**Review by Blackbaud-ErikaMcVey — APPROVED** [review]

**Review by Blackbaud-AdamFunderburk — APPROVED** [review]

**Pull request merged**